tests/unit/parsers/test.manifestjson.js

```diff
@@ -1609,6 +1609,60 @@ describe('ManifestJSONParser', () => {
       expect(errors[0].message).toContain('update_url');
     });
 
+    it('emits a warning for privileged extensions', () => {
+      const addonLinter = new Linter({ _: ['bar'] });
+      const json = validManifestJSON({
+        applications: {
+          gecko: {
+            update_url: 'https://foo.com/bar',
+          },
+        },
+      });
+
+      const manifestJSONParser = new ManifestJSONParser(
+        json,
+        addonLinter.collector,
+        {
+          selfHosted: false,
+          schemaValidatorOptions: {
+            privileged: true,
+          },
+        }
+      );
+
+      expect(manifestJSONParser.isValid).toEqual(true);
+      expect(addonLinter.collector.errors.length).toBe(0);
+      const { warnings } = addonLinter.collector;
+      expect(warnings[0].code).toEqual(messages.MANIFEST_UPDATE_URL.code);
+      expect(warnings[0].message).toContain('update_url');
```

Comment:
given that the rest of the file is using this pattern I think that it is ok to use it here as well for consistency.

In more recent test case I opted to use jest's expect.arrayContains/expect.objectContains to make the test to check just what it cares about and don't fail if there are other warnings logged that the test case doesn't care about, but I think it would be better doing it consistently for the entire test file instead of using a different pattern only on this one.